classes/Directory.js

```diff
@@ -29,11 +29,11 @@ class Directory {
       const endpoint = `${this.baseEndpoint}`
 
       const params = {
-        validateStatus: validateStatus,
         "ref": BRANCH_REF,
       }
 
       const resp = await axios.get(endpoint, {
+        validateStatus: validateStatus,
```

Comment:
This can just be `validateStatus,`

Reply:
> 1. The `validateStatus` function used is replicated in several files - it should exist in a utils file and be imported instead.
> 2. Only the `Directory.js` and `File.js` files have been modified, however `validateStatus` exists also in `Config.js` as part of `params`.
> 3. Finally, `sites.js` needs to implement `validateStatus` in its axios calls

Do we need `validateStatus` in `sites.js`?

Reply:
No we don't, ignore that